sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/init.sql

```diff
@@ -0,0 +1,66 @@
+CREATE OR REPLACE TABLE
+  `mozilla-public-data`.telemetry_derived.agg_active_users_v1
+PARTITION BY
+  DATE(submission_date)
+CLUSTER BY
+  country,
+  app_name,
+  attribution_medium,
+  channel
```

Comment:
should also include option for partition required (at least based on the metadata file that seems to be the intention:
https://github.com/mozilla/bigquery-etl/pull/2952/files#diff-2e37bcde22da96b36b063787a0b603e21093ec0a23a11f2657744b0e530a9f99R19

Here's an example:
https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/core_clients_last_seen_v1/init.sql#L9

Reply:
For active_users_aggregates the partition is defined in metadata.yaml